js/view.js

```diff
@@ -136,8 +139,23 @@ define([
           break;
       }
     },
+    onBlur: function(event) {
+      Adapt.trigger('tooltip:remove');
+    }
```

Comment:
missing some commas

Reply:
Sorry about that, copied this from a class while rushing. Fixed now